.drone/config.yaml

```diff
@@ -0,0 +1,48 @@
+---
+kind: pipeline
+type: docker
+name: default
+
+trigger:
+  ref:
+  - refs/heads/test/**
+  - refs/heads/master
+
+workspace:
+  base: /home/runner/builds
+  path: kubecf
+
+clone:
+  disable: true
+
+# Dockerfile source - https://github.com/f0rmiga/bazel-docker-image.
+.bazel_image: &bazel_image thulioassis/bazel-docker-image:1.2.1
+
+steps:
+- name: clone
```

Comment:
Why do we need custom cloning? It doesn't appear that we are doing anything here that drone's default cloning doesn't handle?

If we need it, please comment on why.

Reply:
Since I moved to use the root user in the image, all the file permissions related to the repo + bazel went away. I also don't think Drone expects us to use any non-root user.